extensions/amp-analytics/0.1/events.js

```diff
@@ -168,6 +188,24 @@ export class CustomEventTracker extends EventTracker {
    * @param {!AnalyticsEvent} event
    */
   trigger(event) {
+    const observers = this.observers_[event.type];
+    if (startsWith(event.type, 'sandbox')) {
```

Comment:
But the method will be one line check. I think `startsWith` is good enough unless we are adding extra logic to check if an event is sandbox event